integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/preaggregate/TestPreAggregateDrop.scala

```diff
@@ -42,14 +42,14 @@ class TestPreAggregateDrop extends QueryTest with BeforeAndAfterAll {
       "create datamap preagg1 on table maintable using 'preaggregate' as select" +
       " a,sum(b) from maintable group by a")
     sql(
-      "create datamap preagg2 on table maintable using 'preaggregate'  as select" +
+      "create datamap preagg2 on table maintable using 'preaggregate' as select" +
       " a,sum(c) from maintable group by a")
-    sql("drop table if exists preagg2")
+    sql("drop datamap if exists preagg2 on table maintable")
     val showTables = sql("show tables")
     checkExistence(showTables, false, "preagg2")
     checkExistence(showTables, true, "preagg1")
   }
-  
+
   test("drop main table and check if preaggreagte is deleted") {
```

Comment:
please add a testcase to drop datamap which is not exist

Reply:
ok